test/mock-dom.js

```diff
@@ -0,0 +1,8 @@
+global.window = {};
+global.document = {
+	createElement: function() {
+		return {
+			tagName: 'iframe'
+		};
+	}
+};
```

Comment:
So the test is already mocking out window and `createElement` (lines 16-21)... maybe instead of importing this file you can use a similar approach?